js/models/messages.js

```diff
@@ -442,7 +442,7 @@
                             // This is primarily to allow the conversation to mark all older messages as
                             //   read, as is done when we receive a read receipt for a message we already
```

Comment:
Maybe we could use a search through the codebase for references to the old terms? 'Read receipt' in the comment refers to 'read sync' now, right?